src/docs/Components/Cards/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Cards'
+order: 1
+description: 'A card is a navigation element that structures a group of related informations that must be understood on its own. It visually looks like a playing card.'
```

Comment:
`a navigation element` : 
I find this mention a bit confusing.
Indeed, in my mind, "element of navigation", it rather refers to a navbar, tabs, a breadcrumb, etc... 

Maybe we can rephrase, or I don't know how to be more specific.